cmd/nerdctl/utils/fmtutil/fmtutil.go

```diff
@@ -45,13 +45,13 @@ func formatLabels(labels map[string]string) string {
 	return res
 }
 
-// formatSlice formats the slice with `--format` flag.
+// FormatSlice formats the slice with `--format` flag.
 //
 // --format="" (default): JSON
 // --format='{{json .}}': JSON lines
 //
-// formatSlice is expected to be only used for `nerdctl OBJECT inspect` commands.
-func formatSlice(cmd *cobra.Command, x []interface{}) error {
+// FormatSlice is expected to be only used for `nerdctl OBJECT inspect` commands.
+func FormatSlice(cmd *cobra.Command, x []interface{}) error {
```

Comment:
If the purpose of the function is formatting rather than printing, I think it should receive io.Writer rather than *cobra.Command.